src/model/state.ts

```diff
@@ -468,7 +469,9 @@ function updateLastPageCommitteeEvents(time: number, snapshot: StateSnapshot, co
 
 function stringArrToObj(arr: string[]): { [k: string]: boolean } {
   const result: { [k: string]: boolean } = {};
-  arr.forEach((value) => {result[value] = true;});
+  arr.forEach((value) => {
```

Comment:
It's strange that Prettier changed this line.. are you running Prettier using 'npm run prettify'?
Because formatting in files you didn't touch shouldn't change since the same Prettier with the same .prettierrc.json should be running..